ci/conda/recipes/morpheus/meta.yaml

```diff
@@ -55,7 +55,7 @@ outputs:
         - ccache
         - ninja
       host:
-        - cuda-python <=11.7.0
+        - cuda-python 11.8
```

Comment:
If we go up to `cuda-python>11.7` then we should specify the version requirements on `cuda-toolkit` since this will now get brought it too.

Reply:
We don't have a direct dependency on `cuda-toolkit`, why is it important that we include it here? That seems like a problem if cuda-python gets more specific about the `cuda-*` packages it depends on.
        - cuda-python 11.8
```